hypha/apply/users/templates/users/activation/email.txt

```diff
@@ -1,8 +1,10 @@
-{% load wagtailadmin_tags %}
+{% load i18n wagtailadmin_tags %}
 {% base_url_setting as base_url %}
-Dear {{ name|default:username }},
+{% name|default:username as user %}
```

Comment:
@Parbhat This seems not to work, suggestions?

I want the translation string below to be the same as in other places to you only need to translate it once.

Reply:
@sks444 or @sandeepsajan0 Suggestion for my issue above?

Reply:
@frjo  I am checking , I think we need to use 'with' to make it available inside a block. Let me check more options.

Reply:
@frjo can you try it like this `{% firstof  name username as user %}`.  I think you wants to show `name` and username only if name is None, right?